docs/quickstart.rst

```diff
@@ -46,38 +46,45 @@ Step 3: Install PRIMO and required dependencies
 
     Opening Anaconda prompt from Start Menu on Windows
 
-* Navigate to the directory where PRIMO source code is extracted. 
-* Create and activate a new environment as follows:: 
-    
-    conda create --name primo python
-    conda activate primo 
+* Navigate to the directory where PRIMO source code is extracted. The path to this directory can be copy pasted from Windows Explorer.
 
-.. figure:: _static/environment.png
+.. figure:: _static/path.png
     :width: 800
     :align: center
 
-    Creating a new 'primo' environment 
+    Obtaining the path where PRIMO source code is downloaded from Windows Explorer
 
-* Install all dependencies with::
- 
-    pip install -r requirements.txt 
-   
-.. figure:: _static/install.png
+* Change active directory in Anaconda prompt with the following command. Please replace the <path> below with the location of the directory copied in the previous step. ::
+
+    cd "<path>"
+
+.. figure:: _static/cd.png
     :width: 800
     :align: center
 
-    Installing PRIMO
+    Changing active directory
 
-* Install the free and open-source solver `SCIP <https://scipopt.org/>`_ using the command below. 
-  Note that PRIMO can also utilize other free or commercial optimization solvers if you have access to other solvers already. ::
+* Create a new "primo" environment with all dependencies installed as follows::
 
-    conda install -c conda-forge -y scip=9.1.0
+    conda env create -f conda-env.yml
 
-.. figure:: _static/scip_install.png
+.. figure:: _static/environment.png
     :width: 800
     :align: center
 
-    Installing SCIP
+    Creating a new 'primo' environment
+
+* Activate the newly created "primo" environment as follows. Note that the command prompt says `(primo)` on the left instead of `(base)` once the environment is successfully activated::
+
+    conda activate primo
+
+.. figure:: _static/activate.png
+    :width: 800
+    :align: center
+
+    Activating the new 'primo' environment
+
+* Note that the the free and open-source solver `SCIP <https://scipopt.org/>`_ will also get installed with the above step. PRIMO can also utilize other free or commercial optimization solvers if you have access to other solvers already.
```

Comment:
Perhaps add a link to where the user can find documentation on how to add a different solver.

Reply:
We don't have a documentation page at the moment... If you agree, we can address that in a later PR